### DISCUSSION TITLE
How can I hide the channel information?

### DISCUSSION THREAD

**Discussion by Junxi-Chen** [Homebrew/discussions#2742] [Everyday usage]
Hi, guys. I have switch the origin channel to other mirror channel to speed up. Yet, when I run 'brew update' it keep showing the information such as 'HOMEBREW_BREW_GIT_REMOTE' which is quite annoying. I am wondering whether there is a way or configuration to hide it. Thanks a lot
![Screen Shot 2022-01-07 at 11 11 19 AM](https://user-images.githubusercontent.com/66880284/148485391-8246f5c7-bd44-4888-8d07-2637e7869a1e.png)
.

**Comment by carlocab** [comment]
Not currently, I'm afraid. I'm not sure we'd accept a pull request that implements a way to silence those messages -- before they were put in place we received a non-negligible number of bug reports caused by usage of a mirror. Since they were put in place, those reports have all but disappeared, so it seems to me that making these as visible as they currently are does help a number of users diagnose issues they're having with Homebrew.
